rpa-builder/6.2/modules/ROOT/pages/toolbox-network-rest-call.adoc

```diff
@@ -1,170 +1,247 @@
 exactly the information that you require without having to use special
-scripting. This simplifies further processing for you as you can access
+scripting, which simplifies further processing as you can access
 the desired result immediately.
++
+The _Json Query_ Action Step enables you to read individual values from the Json object and process the result further.
 
-With the _Json Query_ Action Step, you also have the
-option of reading individual values from the Json object and thus
-specifying the result further.
+** *Request body* 
++
+The request body of the REST call to send 
 
-*Request body* Here you can see what is to be sent.
+** *Request Body Type* 
++
+The request body type of the REST call result
++
+You can choose between _Json_ and _Form URL encoded_.
 
-*Request Body Type* Here you can choose between _JSON_ and _Form URL
-encoded._
+** *Url Extension* 
++
+A path parameter that uniquely identifies the resource
 
-*Url Extension* A path parameter that uniquely identifies the resource.
-
-*XPath* Displays the Xpath expression, for example, /root/access_token/text().
+** *XPath* 
++
+The Xpath expression to the property of the response object
++
+For example, `/root/access_token/text()`.
 
 == Inbound Variables
 
-//link:#AS_RestCall_P_BaseUrl[*Base Url*]
-*Base Url* Here, the end point entered in the wizard is displayed.
-
-//link:#AS_RestCall_P_UrlExtension[*Url Extension*]
-*Url Extension* A path parameter that uniquely identifies the resource.
-
-//link:#AS_RestCall_P_JsonPath[*JsonPath Expression*]
-*Json Path Expression* Displays the JsonPath Expression, such as $.access_token.
-
-//link:#AS_RestCall_P_XPath[*XPath Expression*]
-*XPath Expression* Displays the Xpath expression, for example, /root/access_token/text().
+* *Base Url*
++
+The base URL of the service
+* *Url Extension*
++
+A path parameter that uniquely identifies the resource
+* *Json Path Expression*
++
+The Json path to the property of the response object
++
+For Example, `$.access_token`.
+* *XPath Expression* 
++
+The Xpath expression to the property of the response object
++
+For example, `/root/access_token/text()`.
 
 == Outbound Variables
 
-*IsSuccessful* (boolean) Shows whether the REST Call was successful or not.
+* *IsSuccessful* 
++
+(boolean) Shows whether the REST Call was successful or not
 
-*ResponseBody* (string) Returns the requested information as a string.
+* *ResponseBody* 
++
+(string) Returns the requested information as a string
 
-*FirstItem* (array) The first element of the array is output as a string
+* *FirstItem* 
++
+(array) The first element of the array is output as a string
 
-*Status* (integer) Displays the HTTP status code. 200 for OK, for
-example – the request was processed successfully and the result was
-transferred in the answer
+* *Status* 
++
+(integer) Displays the HTTP status code
++
+For example, `200` is OK, which means the request was processed successfully and the result was

```

Comment:
I have usually seen this written as `200 OK` but I don't know how it shows up in RPA.